PluralKit.Bot/Proxy/ProxyService.cs

```diff
@@ -148,11 +148,21 @@ public bool ShouldProxy(Channel channel, Message msg, MessageContext ctx)
     {
         // Create reply embed
         var embeds = new List<Embed>();
+        var content = "";
+        
         if (trigger.Type == Message.MessageType.Reply && trigger.MessageReference?.ChannelId == trigger.ChannelId)
         {
             var repliedTo = trigger.ReferencedMessage.Value;
             if (repliedTo != null)
             {
+                if (trigger.Mentions.Length > 0 
+                    && repliedTo.Author.Id == trigger.Mentions[0].Id
+                    && !(trigger.Content.Contains($"<@{repliedTo.Author.Id}>")
+                         || trigger.Content.Contains($"<@!{repliedTo.Author.Id}>")))
+                {
+                    content = $"*<@{repliedTo.Author.Id}>*\n";
```

Comment:
```suggestion
                    content = $"<@{repliedTo.Author.Id}>\n";
```

Italics around an @-mention do nothing afaik, and it wastes 2 characters

Reply:
It does on mobile

![Screenshot_20230714-140950~2.png](https://github.com/PluralKit/PluralKit/assets/48085877/61383e35-fb25-4d90-81d0-ecf602ee40c9)

I caj't remove it easily on mobile but I can get to it when I'm at home, if it should be removed

Reply:
I can't see it being needed either way

Reply:
I don't see a reason to remove it, can you explain why you would?